<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chartsWindow</class>
 <widget class="QDialog" name="chartsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="comeName">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Charts Window</string>
   </property>
  </widget>
  <widget class="QLabel" name="arr1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>620</y>
     <width>411</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>arr1</string>
   </property>
  </widget>
  <widget class="QLabel" name="arr2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>620</y>
     <width>481</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>arr2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showArr">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>620</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Arrays</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showChart">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Show chart</string>
   </property>
  </widget>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>711</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
